src/v/config/configuration.cc

```diff
@@ -99,6 +99,19 @@ configuration::configuration()
       "Interval for which all coprocessor offsets are flushed to disk",
       required::no,
       300000ms) // five minutes
+  , enable_v8(*this, "enable_v8", "Enable v8 engine", required::no, false)
+  , core_for_executor(
+      *this,
+      "core_for_executor",
+      "Core for std::thread in executor",
```

Comment:
add info about what is core?

Reply:
You can use the same description as for `cpuset` parameter in `reactor.cc`